api/Request.json

```diff
@@ -11,6 +11,10 @@
           "chrome_android": {
             "version_added": "42"
           },
+          "deno": {
+            "version_added": "1.0",
+            "notes": "Fetching with a <code>\"manual\"</code> redirect mode does not result in a <code>\"opaqueredirect\"</code> response, but a regular response."
```

Comment:
Perhaps this note could go on the constructor (the next feature down), instead?

Reply:
Done